distributor-api-v1/operations.md

```diff
@@ -1263,12 +1265,12 @@ In case of an error caused by insufficient availability \(which might have decre
 }
 ```
 
-| Property | Type | Description |
-| :--- | :--- | :--- |
-| `Currency` | string | ISO 4217 code of the currency. |
-| `GrossValue` | number | Gross value of the amount. (Net + sum of TaxValues) |
-| `NetValue` | number | Net value of the amount. |
-| `TaxValues` | array of [TaxValue](operations.md#taxvalue)s | Tax values of the amount. |
+|  | Property | Type | Description |
```

Comment:
why is there double pipe? '|'

Reply:
Just to make the file consistent
![image](https://user-images.githubusercontent.com/6573164/116700735-656a8d80-a9c7-11eb-94d5-5209f32d3264.png)

But next PR will be in terms of consolidation and having it this way:
![image](https://user-images.githubusercontent.com/6573164/116700997-afec0a00-a9c7-11eb-9c18-6e0f7abfaa24.png)

I just did not noticed before and I would like to not mix new docs with refactoring current state